k8s/chart/templates/deployment.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "sdp-operator.fullname" . }}
+  namespace: {{ .Release.Namespae }}
+  labels:
+  {{- include "sdp-operator.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "sdp-operator.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "sdp-operator.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.sdp.operator.image.pullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      serviceAccountName: {{ include "sdp-operator.serviceAccountName" . }}
+      containers:
+        - name: {{ .Chart.Name }}
+          env:
+            - name: APPGATE_OPERATOR_NAMESPACE
+              value: {{ .Release.Namespace }}
+            - name: APPGATE_OPERATOR_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "sdp-operator.secret" . }}
+                  key: appgate-operator-user
+            - name: APPGATE_OPERATOR_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "sdp-operator.secret" . }}
+                  key: appgate-operator-password
+            - name: APPGATE_OPERATOR_HOST
+              value: {{ required "A valid .Values.sdp.operator.host entry is required!" .Values.sdp.operator.host }}
+            - name: APPGATE_OPERATOR_DEVICE_ID
+              value: {{ required "A valid .Values.sdp.operator.deviceId entry is required!" .Values.sdp.operator.deviceId }}
+            - name: APPGATE_OPERATOR_LOG_LEVEL
+              value: {{ .Values.sdp.operator.logLevel }}
+            - name: APPGATE_OPERATOR_TIMEOUT
+              value: {{ .Values.sdp.operator.timeout }}
+            - name: APPGATE_OPERATOR_TARGET_TAGS
+              value: {{ .Values.sdp.operator.excludeTags }}
+            - name: APPGATE_OPERATOR_EXCLUDE_TAGS
+              value: {{ .Values.sdp.operator.excludeTags }}
+            - name: APPGATE_OPERATOR_BUILTIN_TAGS
+              value: {{ .Values.sdp.operator.builtinTags }}
+            - name: APPGATE_OPERATOR_DRY_RUN
+              value: {{ .Values.sdp.operator.dryRun }}
+            - name: APPGATE_OPERATOR_CLEANUP
+              value: {{ .Values.sdp.operator.cleanup }}
+            - name: APPGATE_OPERATOR_TWO_WAY_SYNC
+              value: {{ .Values.sdp.operator.twoWaySync }}
+            - name: APPGATE_OPERATOR_SSL_NO_VERIFY
+              value: {{ .Values.sdp.operator.sslNoVerify }}
+            - name: APPGATE_OPERATOR_CONFIG_MAP
+              value: {{ .Values.sdp.operator.configMapMt }}
+            {{ with .Values.sdp.operator.caCert }}
+             - name: APPGATE_OPERATOR_CACERT
+               value: {{ . }}
```

Comment:
There seem to be an extra space indented here...

Reply:
right